### ISSUE TITLE
Question : Can I add a comment to the post?

### ISSUE THREAD

**Issue by cartoonpoet** [jeffreytse/jekyll-theme-yat#40]
Hello!

I'm a college student who is using the blog that Jeffreytse made very well.

After writing the post, I want to add it so that visitors can write comments at the bottom, and I want to know if you provide this feature!

**Comment by jeffreytse** [comment]
Hi @cartoonpoet 

Sure, this theme has provided the comment feature (Disqus, Gitment), and you need to config it by `_config.yml`.

You can have a glance at my [blog](https://jeffreytse.net), and welcome to star this project for further updates in the future. : )

![image](https://user-images.githubusercontent.com/9413601/118999140-96cdec00-b9bc-11eb-92b6-921e6bcaae4d.png)

Thanks and regards

**Comment by cartoonpoet** [comment]
Thank you! 
I'll give it a try.

**Comment by DasolU** [comment]
Hi :-) Thank you for this amazing blog theme. I love it so much!
I also having a trouble making gitment work.
I tried to add _config.yml like below image
![image](https://user-images.githubusercontent.com/80796260/121022112-03c6eb80-c7dd-11eb-9478-cfffa9531438.png)
However my blog still doesn't work like below image
![image](https://user-images.githubusercontent.com/80796260/121021821-bc405f80-c7dc-11eb-9a6f-7ceb8e4534ac.png)
To solved the problem, I googled it for 1day but I failed to fix it..
I know this is a vague question.
But I am new to front-end, can anybody help me?
Thank you all!
